lib/game_stats.rb

```diff
@@ -1,4 +1,70 @@
 
 
+
+
 module GameStats
+
+	def highest_total_score # integer
+		all_scores.sort.pop
+	end
+
+	def lowest_total_score # integer
+		all_scores.sort.shift
+	end
```

Comment:
Really like the method chaining here. Could have maybe used a .last and .first (but they'll give you the same return value so it doesn't really matter)